config/routes.rb

```diff
@@ -28,6 +28,7 @@
   match '/api/twilio/voice' => 'twilio_voice#show',
         via: %i[get post],
         format: :xml
+  post '/api/verify/images' => 'idv/image_uploads#create'
```

Comment:
Sure, good call

Reply:
Since the router is drawn once, flipping feature flags is hard to test, and our default flag for this is off

So I ended up moving 404 logic in to the controller, similar to how we do for PIV/CAC stuff:

7fec5886c3149d7faa187bb93d30eac399d7c3ea

Prior art: https://github.com/18F/identity-idp/blob/master/app/controllers/idv/cac_controller.rb#L5